code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -384,10 +388,11 @@
 	fire_delay = 3
 	burst_shot_delay = 2.2
 	is_automatic = TRUE
+	extra_damage = -5
 	automatic = 1
-	slowdown = 0.3
-	autofire_shot_delay = 2
-	spread = 16
+	autofire_shot_delay = 1.70
+	recoil - 1
```

Comment:
...You forgot an = here as well. Unless this is a new change, either way- "recoil = -1"

Reply:
Thanks for the heads up